### ISSUE TITLE
[MAS1.4.3] Luminosity Contrast Ratio of links present on Welcome screen (in blue) is less than required 4.5:1.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#758]
**Actual Result:**
Luminosity contrast ratio of links present on Welcome screen is 4.1:1.

**Expected Result:**
Luminosity contrast ratio of links present on Welcome screen should be greater than or equal to 4.5:1.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Notice links present on Welcome screen (in blue).
3. Open 'Colour Contrast Analyser' application.
4. Select Foreground and Background color of mentioned text.
5. Check value of 'Luminosity Contrast Ratio'.

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 1.4.3 – Contrast (Minimum) (17)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/Doc.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}&action=view&wdAccPdf=0&wdparaid=ABCCC12)

**Attachment:**

![mas1 4 3_botframeworkemulator_welcomescreen_links](https://user-images.githubusercontent.com/41668935/44018500-a104b2e8-9ef9-11e8-8be5-1c3d9eeb3a5e.png)

**Comment by corinagum** [comment]
Hi @mewa1024 Could you let me know what this color should be updated to? Current hex value is: #4672DA

**Comment by corinagum** [comment]
Related to issue [725](https://github.com/Microsoft/BotFramework-Emulator/issues/725);
